diesel.toml

```diff
@@ -3,3 +3,6 @@
 
 [print_schema]
 file = "src/schema.rs"
+with_docs = true
+import_types = ["diesel::sql_types::*"]
+patch_file = "src/schema.patch"
```

Comment:
Not quite sure what to use this for. Can't we see how the schema is changing in the migrations?

Reply:
@svenstaro its easy too look database changes in 1 file rather than checking git history of entire migration files.

Reply:
So this file contains the information of the last migration that took place? Will it change for every migration? I don't believe I quite grasp what this file is supposed to do.

Reply:
it specifies a .patch file to be applied to your schema after it is generated.

Reply:
Sorry, I just don't see a use case why we would always want to point this to `src/schema.patch`. Can you draft up a scenario?

Reply:
@svenstaro see this https://github.com/diesel-rs/diesel/issues/2154 . Yea it may be case it may not be case :)

Reply:
Anyway if you don't like u can remove that line and move ahead :)